lib/pretty_text/shims.js

```diff
@@ -19,6 +19,9 @@ __setUnicode = function(replacements) {
         replacement = "\u200b" + replacement;
       }
       text = text.replace(m[0], replacement);
+      // fixes Safari VARIATION SELECTOR-16 issue with some emojis
+      // https://meta.discourse.org/t/emojis-selected-on-ios-displaying-additional-rectangles/86132
+      text = text.replace('\ufe0f', '');
```

Comment:
Do we want to run that in the loop? Can't it be done once before the loop?

Reply:
You're of course right, I moved it outside the loop.